pennylane/devices/tests/__init__.py

```diff
@@ -97,8 +97,7 @@ def get_device_tests():
 
 def test_device(
     device_name,
-    analytic=None,
-    shots=None,
+    shots=0,
```

Comment:
Note: changed to `shots=0` because `shots=None` is now ambiguous. There's no distinction between `None` being the default and `None` passed explicitly.